<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>345</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>frames</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>titleicon.ico</normaloff>titleicon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextBrowser" name="fileText">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>421</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>7</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="LoadBtn">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>60</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>foldericon.ico</normaloff>foldericon.ico</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="staBtn">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>230</y>
      <width>91</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="colBtn">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>230</y>
      <width>101</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Pictures</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="RechBox">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>120</y>
      <width>171</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Resize Picture</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="wetLine">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>170</y>
      <width>91</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="hetLine">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>170</y>
      <width>91</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="mulLal">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>170</y>
      <width>16</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
